pkg/doc/verifiable/credential_jwt.go

```diff
@@ -0,0 +1,260 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package verifiable
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/square/go-jose/v3"
+	"github.com/square/go-jose/v3/jwt"
+)
+
+// JWTCredClaims is JWT Claims extension by Verifiable Credential.
+type JWTCredClaims struct {
+	*jwt.Claims
+
+	Credential *rawCredential `json:"vc,omitempty"`
+}
+
+// MarshalSigned marshals JWT into signed form (JWT)
+func (vcc *JWTCredClaims) MarshalSigned(signatureAlg JWTAlgorithm, privateKey interface{}, keyID string) (string, error) { //nolint:lll
```

Comment:
Do I understand right that MarshalSigned produces a JWS and MarshalUnsecured produces an UnsecuredJWT?

Reply:
Yes.